frontend/public/components/persistent-volume-claim.jsx

```diff
@@ -60,20 +64,18 @@ const menuActions = [
 ];
 
 export const PVCStatus = ({ pvc }) => {
-  const pvcStatusExtensions = useExtensions(isPVCStatus);
+  const [pvcStatusExtensions] = useExtensions(isPVCStatus);
   if (pvcStatusExtensions.length > 0) {
     const sortedByPriority = pvcStatusExtensions.sort(
       (a, b) => b.properties.priority - a.properties.priority,
     );
     const priorityStatus = sortedByPriority.find((status) => status.properties.predicate(pvc));
+    const PriorityStatusComponent = priorityStatus?.properties?.status;
```

Comment:
The `status` component is loaded via `CodeRef` so we need to use `useResolvedExtensions` (?)